apis/v1alpha2/policy_types.go

```diff
@@ -23,16 +23,10 @@ package v1alpha2
 // for Gateway API.
 type PolicyTargetReference struct {
 	// Group is the group of the target resource.
-	//
-	// +kubebuilder:validation:MinLength=1
-	// +kubebuilder:validation:MaxLength=253
-	Group string `json:"group"`
+	Group Group `json:"group"`
 
 	// Kind is kind of the target resource.
-	//
-	// +kubebuilder:validation:MinLength=1
-	// +kubebuilder:validation:MaxLength=253
-	Kind string `json:"kind"`
+	Kind Kind `json:"kind"`
```

Comment:
rant: Why can't github highlight the specific text that has changed in this line.